plugins/schedule/base/lib/model/LiveStreamScheduleEvent.php

```diff
@@ -107,4 +124,10 @@ public function getCalculatedStartTime()
 		return parent::getCalculatedStartTime() - $preStartTime;
 	}
 
+	public function getCalculatedEndTime()
+	{
+		$postEndTime = !is_null($this->getPostEndTime()) ? $this->getPostEndTime() : 0;

```

Comment:
All the test is  useless , you can just make sure that getPostEndTime return default value of 0